Pinta.Core/Effects/BaseEffect.cs

```diff
@@ -84,35 +84,12 @@ public abstract class BaseEffect
 	/// Launches the configuration dialog for this effect/adjustment.
 	/// If IsConfigurable is true, the ConfigDialogResponse event will be invoked when the user accepts or cancels the dialog.
 	/// </summary>
-	public virtual void LaunchConfiguration ()
```

Comment:
What do you think about making the return type a `Task<bool>`, or some wrapper around a bool to make the meaning more clear, sort of like  the `ConfigDialogResponseEventArgs` from before?

My main thoughts are
- `Gtk.ResponseType` has a lot of values in the enum, so it's less clear which values we care about or are possible to have to deal with. It's really just success / failure that matter
- Ideally the effect API should be as independent as possible from UI stuff like GTK. Using the `ResponseType` enum probably doesn't cause anything bad like forcing GTK libraries to be loaded / initialized for the unit tests, but it does mean that any API changes when upgrading to future GTK versions have more of an impact on the effects

Reply:
I think `Task<bool>` is good for now while the API is being defined. I have changed it now